msticpy/common/pkg_config.py

```diff
@@ -25,14 +25,17 @@
 import yaml
 from yaml.error import YAMLError
 
-from .utility import MsticpyConfigException
+from .utility import MsticpyConfigException, is_valid_uuid
 from .._version import VERSION
 
 __version__ = VERSION
 __author__ = "Ian Hellen"
 
 _CONFIG_FILE: str = "msticpyconfig.yaml"
 _CONFIG_ENV_VAR: str = "MSTICPYCONFIG"
+_DP_KEY = "DataProviders"
+_AZ_SENTINEL = "AzureSentinel"
+_AZ_CLI = "AzureCLI"
```

Comment:
If we are hardcoding this should we update our msticpy.rst file example to include this?

Reply:
Done. I;ve documented the AzureCLI as going in the DataProviders section. It can actually go in either. I;ve updated the config validation to account for it being in either place.